themes/dpla/css/main.css

```diff
@@ -627,7 +627,6 @@ line-height: 14px;
         }
             .thumbs-list li {
                 display: inline-block;
-                float: left;
```

Comment:
This `float` property was overriding the natural behavior of `display: inline-block`, which produces the desired behavior.